### PR TITLE
Added Kaazing Gateway

### PR DESCRIPTION
**Pull request by dpwspoon**
The pull request for the official image is [here](https://github.com/docker-library/official-images/pull/806)

### CODE REVIEW

File: kaazing-gateway/content.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# What is the KAAZING Gateway?
+
+The Kaazing Gateway is a network gateway created to provide a single access point for real-time web based protocols that supports load balancing, clustering, and security management. It is designed to provide scalable and secure bidirectional event-based communication over the web; on every platform, browser, and device.
+
+![logo](https://raw.githubusercontent.com/docker-library/docs/master/kaazing-gateway/logo.png)
```
Comment (tianon) [inline kaazing-gateway/content.md]: This should just be `%%LOGO%%` -- `update.sh` will do the rest. :+1:

**Comment by tianon** [comment]
rest of the content looks solid :+1:

File: kaazing-gateway/content.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Up and Running
+
+By default the gateway runs a WebSocket echo service similar to [websocket.org](https://www.websocket.org/echo.html).
+
+	docker run --name some-kaazing-gateway -h somehostname -d -p 8000:8000 kaazing-gateway
+
+You should then be able to connect to ws://somehostname:8000 from the [WebSocket echo test](https://www.websocket.org/echo.html).
+
+Note: this assumes that `somehostname` is resolvable from your browser, you may need to add and etc/hosts entry for `somehostname` on your `dockerhost ip`.
```
Comment (yosifkit) [inline kaazing-gateway/content.md]: `s/add and/add an/`

**Comment by yosifkit** [comment]
LGTM

**Comment by tianon** [comment]
LGTM

**Pull request merged**